sei/chain.json

```diff
@@ -0,0 +1,118 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "sei",
+    "status": "live",
+    "website": "https://www.sei.io/",
+    "network_type": "mainnet",
+    "pretty_name": "Sei",
+    "chain_id": "pacific-1",
+    "bech32_prefix": "sei",
+    "daemon_name": "seid",
+    "node_home": "$HOME/.sei",
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "usei",
+                "fixed_min_gas_price": 0,
+                "low_gas_price": 0,
+                "average_gas_price": 0.025,
+                "high_gas_price": 0.04
+            }
+        ]
+    },
+    "staking": {
+        "staking_tokens": [
+            {
+                "denom": "usei"
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/sei-protocol/sei-chain",
+        "recommended_version": "v3.0.0",
+        "compatible_versions": [
+            "v3.0.0"
+        ],
+        "cosmos_sdk_version": "",
+        "cosmwasm_enabled": true,
+        "cosmwasm_path": "$HOME/.sei/wasm",
+        "genesis": {
+            "genesis_url": ""
```

Comment:
```suggestion
            "genesis_url": "https://github.com/sei-protocol/testnet/blob/main/pacific-1/genesis.json"
```

Reply:
note: we likely will need to rename the repo from the original name of `testnet` to better reflect that it has mainnet stuff too, will probably name it like `networks` or something, but @udpatil to follow up on that